### DISCUSSION TITLE
Submission package validation often returns empty errors

### DISCUSSION THREAD

**Discussion by dankolbman** [nih-cfde/published-documentation#209] [Bug] [answered]
Often I retrieve the following error:
<img width="669" alt="Screen Shot 2021-03-25 at 9 17 37 PM" src="https://user-images.githubusercontent.com/2495894/112645659-17d59080-8e1d-11eb-85a5-7a2ab817d838.png">

This usually relates to formatting of one of the files being incorrect (bad newline characters, for example), but there are also instances where I am simply not able to figure out why the error is being thrown.

**Comment by ACharbonneau** [comment]
Thanks! This is related to the issues seen here https://github.com/nih-cfde/published-documentation/discussions/142 and here https://github.com/nih-cfde/published-documentation/discussions/184#discussioncomment-531186 

When there is an error that is something like "your table is missing a header" or "a header is misspelled" or "a table name is misspelled", essentially, anything that looks like schema information is missing it gives this very unhelpful error instead of telling you which thing is missing.

I already have the team working on trying to pull out what the actual problem is and display it and I'm hoping we'll have improved error handling for missing info in the next release. For the moment, if you send me the zip file (over email or slack or whatever) I can diagnose the specific issue for you.

**Reply by ericwenger-pm** [reply]
Hi, Amanda, I'm picking this up again from our lead, also encountered this error.  I'll provide to you our file and the terminal output.

**Comment by ACharbonneau** [comment, marked as answer]
We have been correcting poor error messages as we find them. Please continue to report similar issues in this thread as they arise.
